docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -97,164 +83,204 @@ can be seen in the Information tab, as shown in the image below. For more inform
+  * **PWM**: This mode means that the module will apply a fraction of the battery voltage as its drive voltage when given a setpoint. For example, if your battery voltage is 20V, and you send a 50% command, then the module will apply a 10V drive voltage. 
     
-    * Despite the similar naming, this mode has nothing to do with using the Hobby PWM Communication Protocol
-  
-  * Voltage: This mode interprets the commands as a fraction of the maximum voltage set in the Tuning tab. So if your maximum voltage was set at 8V, and you sent a 25% throttle command, the motor would apply a drive voltage of 2V.
-  * Velocity: This mode interprets the command as a fraction of the maximum velocity set in the Tuning tab. So if your maximum velocity was set at 100 rad/s, and you sent a 25% throttle command, the motor would try to spin at 25 rad/s.
+    * Despite the similar naming, this mode has nothing to do with using the Standard PWM hobby protocol
   
-  For this setup, you should **set this to Voltage mode**.
+  * **Voltage**: This mode interprets the commands as a fraction of the maximum voltage set in the Tuning tab. So if your maximum voltage was set at 8V, and you sent a 25% throttle command, the module would apply a drive voltage of 2V.
+  * **Velocity**: This mode interprets the command as a fraction of the maximum velocity set in the Tuning tab. So if your maximum velocity was set at 100 rad/s, and you sent a 25% throttle command, the module would try to spin at 25 rad/s.
 
-* **Motor Direction**: This sets what direction the motor considers to be the positive direction for throttle commands from a flight controller. 
-  So in 2D mode, like we are for this example, the motor will always spin this way on a throttle command.  The use of 2D or 3D should match the "FC 2D/3D Mode" parameter, 
-  which defaults to 2D. **Set this to 2D Counter Clockwise**. 
-  
-Any other parameters in this tab can be safely left at their default values.
+The meaning of this parameter is covered in greater detail in the :ref:`throttle_mode_maximums_directions` section of the Feature Reference Manual. 
+
+For this setup, you should **put the module in Voltage mode**. To do that, set the *Mode* parameter to *Voltage* in Control Center, as shown below.
+
+.. figure:: ../_static/tutorial_images/pwm_flight_controller/control_center_mode.png
+    :align: center
+
+    Mode Parameter
+
+Motor Direction
+################
+This sets what direction the module considers to be the positive direction for throttle commands from a flight controller.  When the *Motor Direction* parameter is
+set to either of the 2D directions (*2D clockwise* or *2D counter clockwise*), the module will always spin in the specified direction on a throttle command. For example,
```

Comment:
either counterclockwise or counter-clockwise I think

Reply:
The exact name of the parameter in IQ Control Center, like how it appears in the dropdown, is 2D counter clockwise, with the space. The correct thing to do is counterclockwise or counter-clockwise, but when am I referring to the actual parameter, I use its exact spelling in the IQ Control Center, regardless of correctness. I want to reflect the parameter name